.babelrc.js

```diff
@@ -10,5 +10,7 @@ module.exports = {
   ],
   plugins: [
     '@babel/plugin-proposal-function-bind',
+    // `loose:true` is fine because we don't use document.all
+    ['@babel/plugin-proposal-nullish-coalescing-operator', { loose: true }],
```

Comment:
This is already included in the latest `@babel/preset-env`, so we just need an upgradation.

Reply:
Rebased.